position/positions.go

```diff
@@ -35,27 +35,27 @@ func AltDotEnd(buff demodel.CharBuffer) (uint, error) {
 
 func TagDotStart(buff demodel.CharBuffer) (uint, error) {
 	if buff.Tagline == nil || len(buff.Tagline.Buffer) == 0 {
-		return 0, Invalid
+		return 0, ErrInvalid
```

Comment:
I wonders if it would be useful to have more context provided in some of these uses of `Invalid`, but I don't know this file well enough to be sure.